pkg/commands/options/publish.go

```diff
@@ -65,6 +66,13 @@ type PublishOptions struct {
 	BaseImportPaths bool
 	// Bare uses a tag on the KO_DOCKER_REPO without anything additional.
 	Bare bool
+	// ImageNameSeparator is used to create the image name from base repo and
+	// specific import path. If not set a default of "/" will be used.
+	ImageNameSeparator string
```

Comment:
Do we still need this now that the flag is dropped and you can just override `ImageNamer`?

Reply:
Hmm. I could drop this too.

But, I thought it would be easier to just allow to change separator and still use built in algorithms.

Reply:
I did remove the `ImageNameSeparator` option